target/linux/qualcommax/ipq60xx/base-files/etc/board.d/01_leds

```diff
@@ -0,0 +1,20 @@
+
+. /lib/functions/uci-defaults.sh
+
+board_config_update
+
+board=$(board_name)
+
+case "$board" in
+linksys,mr7350)
+	ucidef_set_led_netdev "lan1-port-link" "LAN1-PORT-LINK" "90000.mdio-1:00:green:lan" "lan1" "tx rx link_10 link_100 link_1000"
+	ucidef_set_led_netdev "lan2-port-link" "LAN2-PORT-LINK" "90000.mdio-1:01:green:lan" "lan2" "tx rx link_10 link_100 link_1000"
+	ucidef_set_led_netdev "lan3-port-link" "LAN3-PORT-LINK" "90000.mdio-1:02:green:lan" "lan3" "tx rx link_10 link_100 link_1000"
+	ucidef_set_led_netdev "lan4-port-link" "LAN4-PORT-LINK" "90000.mdio-1:03:green:lan" "lan3" "tx rx link_10 link_100 link_1000"
+	ucidef_set_led_netdev "wan-port-link" "WAN-PORT-LINK" "90000.mdio-1:04:green:wan" "wan" "tx rx link_10 link_100 link_1000"
```

Comment:
This does not match the stock firmware.
Also you need to configure yellow led.

Reply:
Done if I'm not misreading the code...

Reply:
If I'm not mistaken, it should be like this:
```
ucidef_set_led_netdev "wan-port-activity" "WAN-PORT-ACTIVITY" "...amber:wan" "wan" "tx rx"
ucidef_set_led_netdev "wan-port-link" "WAN-PORT-LINK" "...green:wan" "wan" "link_10 link_100 link_1000"
```